Update payment partners 2024 q1

chihsuan opened the pull request:
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

Closes https://github.com/Automattic/woocommerce.com/issues/19237


1. Removed ZipCo from all countries
2. Removed Eway from APAC (all countries since Eway was only added to APAC countries)
3. Added new partner `AirWallex` ,  located right below `Paypal` for EU+UK, located below `Stripe` and above `Paypal` for APAC
    - Title: `Airwallex Payments`
    - Description: `Boost international sales and save on FX fees. Accept 60+ local payment methods including Apple Pay and Google Pay.`
    - Countries: `[ 'GB', 'AT', 'BE', 'EE', 'FR', 'DE', 'GR', 'IE', 'IT', 'NL', 'PL', 'PT', 'AU', 'NZ', 'HK', 'SG', 'CN' ]`
4. Fixed lint `Short array syntax is not allowed` errors.

- ~Re-order `Mercado Pago` in LATAM. It should now be above PayPal~  **(It's already above PayPal)**
 

<!-- Begin testing instructions -->

### How to test the changes in this Pull Request:

<!-- Include detailed instructions on how these changes can be tested. Review and follow the guide for how to write high-quality testing instructions. -->

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start with a fresh install
1. Go to core profiler
1. Choose `UK` as store country
1. Complete core profiler without installing any plugins
1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com` > **turn off** "Show Suggestions" checkbox
1. Go to `WooCommerce -> Home` and choose the `Set up payments` task.
1. Confirm it shows Stripe, PayPal Payments, Airwallex Payments, and Mollie in order under the "Other payment providers" section.
1. Confirm you can install Airwallex Payments via payment task
1.  Change store country to Australia
1. Go to `Set up payments` task.
1. Confirm it shows Stripe, Airwallex Payments and PayPal Payments in order under "Other payment providers" section. (No Eway)
1. Install WCpay
1. Go to `Tools > WCA Test Helper > REST API FIlters` (/wp-admin/tools.php?page=woocommerce-admin-test-helper)

```
Click new filters:

- Endpoint: /wc/v3/payment_gateways
- Dot Notation: 0.id
- Replacement: woocommerce_payments

- Endpoint: /wc/v3/payment_gateways
- Dot Notation: 0.needs_setup
- Replacement:false

- Endpoint: /wc/v3/payment_gateways
- Dot Notation: 0.enabled
- Replacement: true
```

15. You should see the `Additional payment options` section as follows (no ZipCo)

![Screenshot 2024-01-11 at 20 21 50](https://github.com/Automattic/woocommerce.com/assets/4344253/04dd06a6-4af7-485d-bb96-5188827a7aac)

16. Please test around

<!-- End testing instructions -->

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box and supplying data. -->

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

<!-- Choose only one -->

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message <!-- Add a changelog message here -->

Update onboarding payment partners 

#### Comment <!-- If the changes in this pull request don't warrant a changelog entry, you can alternatively supply a comment here. Note that comments are only accepted with a significance of "Patch" -->

</details>

chihsuan committed:
Remove eway and zipco

chihsuan committed:
Add Airwallex

Commit:
Add changefile(s) from automation for the following project(s): woocommerce

chihsuan committed:
Fix lint

chihsuan committed:
Update airwallex images

ilyasfoo commented on plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php:
```diff
@@ -177,23 +190,8 @@ public static function get_all() {
-					'AU',
-				),
+				'is_visible'          => false,
+				'category_other'      => false,
```
Not sure if this is intended, but comparing to the wccom change (https://github.com/Automattic/woocommerce.com/pull/19279/files#diff-9606b09b90678ea8f6e59df883aeb7e7dff2bee0dbeb065eca6f147cc5461237R206), it's `array()`

chihsuan replied:
Yeah, good catch! It should be an array. Fixed in ec23dca52fe8ba4d0d256ade76278b0a7837225d

ilyasfoo reviewed (approved):
Tested well with minor comment, LGTM!

chihsuan committed:
Fix category_other value

rjchow reviewed (approved):
Looks good for France! 

(sidenote, I wonder if we shouldn't be posting the google sheets link on a public repo? it's access-controlled but just in case someone messes that up at some point)

chihsuan commented:
> (sidenote, I wonder if we shouldn't be posting the google sheets link on a public repo? it's access-controlled but just in case someone messes that up at some point)

Good point! Removed it from the instructions and deleted the editing history.
